src/containers/Cfe/index.tsx

```diff
@@ -128,103 +130,146 @@ class Cfe extends React.Component<CfeProps, CfeState> {
             styles.content,
             { [styles.hidden]: currentTab !== VIEWER_TAB_KEY },
         ]);
-        const plotClassNames = classNames([{ [styles.hidden]: currentTab === VIEWER_TAB_KEY }]);
+        const plotClassNames = classNames([
+            styles.content,
+            { [styles.hidden]: currentTab === VIEWER_TAB_KEY },
+        ]);
 
-        return (
+        type MenuItem = Required<MenuProps>["items"][number];
```

Comment:
Looks like in `GalleryCard` you were able to `import { ItemType } from "antd/es/menu/interface";`. Is that the same type you need here?